CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- Changing controller `httpEndpoint` to `8610` because of overlapping ports.
```

Comment:
Do you mean `8160`?

Reply:
Sorry my bad, I made a mistake in the httpEndpoint. It is actually `8610`. Good catch ❤️